docs/docs/guides/integrate/zitadel-apis/access-zitadel-apis.md

```diff
@@ -0,0 +1,237 @@
+:::
+
+For further details and in-depth exploration, you can refer to the [ZITADEL API documentation](/docs/apis/introduction).
+
+## How to access ZITADEL APIs
+
+Accessing ZITADEL APIs, except for the Auth API and the System API, requires these basic steps:
+
+1. **Create a service user**: A service user is a special type of account used to grant programmatic access to ZITADEL's functionalities. Unlike regular users who log in with a username and password, [service users rely on a more secure mechanism involving digital keys and tokens](../service-users/authenticate-service-users).
+2. **Give permission to access ZITADEL APIs**: Assign a Manager role to the service user, giving it permission to make changes to certain resources in ZITADEL.
+3. **Authenticate the service user**: Like human users, service users must authenticate and request a OAuth token with the scope `urn:zitadel:iam:org:project:id:zitadel:aud` to access ZITADEL APIs. [Service users can be authenticated](../service-users/authenticate-service-users) using private key JWT, client credentials, or personal access token.
+4. **Access ZITADEL APIs with the token**: The OAuth token must be included in the Authorization Header of calls to ZITADEL APIs.
+
+### Accessing Auth API
+
+The Auth API can be used for all operations on the requesting user, meaning the user id in the sub claim of the used token.
+Using this API doesn't require a service user to be authenticated.
+Instead you call the Auth API with the token of the user.
+
+[Reference documentation for authentication API](/docs/apis/introduction#authentication)
+
+### Accessing System API
+
+With the System API developers can manage different ZITADEL instances.
+The System API can't be accessed with service users and requires a special configuration and authentication that can be found in our [guide to access ZITADEL's System API](./access-zitadel-system-api).
+
+[Reference documentation for system API](/docs/apis/introduction#system)
+
+## 1. Create a service user
+
+First you need to create a new service user through the console or ZITADEL APIs.
+
+Via Console:
+
+1. In an organization, navigate to Users > Service Users
+2. Click on **New**
+3. Enter a username and a display name
+4. Click on **Create**
+
+Via APIs:
+
+* [Create User (Machine)](/docs/apis/resources/mgmt/management-service-add-machine-user)
+
+## 2. Grant a Manager role to the service user
+
+ZITADEL Managers are Users who have permission to manage ZITADEL itself.
+There are some different levels for managers.
+
+- **IAM Managers**: This is the highest level. Users with IAM Manager roles are able to manage the whole instance.
+- **Org Managers**: Managers in the Organization Level are able to manage everything within the granted Organization.
+- **Project Mangers**: In this level the user is able to manage a project.
+- **Project Grant Manager**: The project grant manager is for projects, which are granted of another organization.
```

Comment:
```suggestion
- **Project Grant Manager**: The project grant manager is for projects, which are granted to another organization.
```

Reply:
I think of is right, as it is when you got the project from another org